From faef126255222f727219bf69ac2b89857dc61ab2 Mon Sep 17 00:00:00 2001
From: Nate Graham <nate@kde.org>
Date: Sun, 14 Jun 2020 09:08:21 -0600
Subject: [PATCH] Port applets to use PlasmaExtras.PlaceholderMessage

The Kirigami version uses QQC2 elements which don't respect the plasma
theme and color scheme, leading to thing slooking out of place
especially when using a dark color theme or scheme.

Depends on https://invent.kde.org/frameworks/plasma-framework/-/merge_requests/13
---
 applets/clipboard/contents/ui/Menu.qml                          | 2 +-
 .../notifications/package/contents/ui/FullRepresentation.qml    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/applets/clipboard/contents/ui/Menu.qml b/applets/clipboard/contents/ui/Menu.qml
index 0bd6064a5..cac8380db 100644
--- a/applets/clipboard/contents/ui/Menu.qml
+++ b/applets/clipboard/contents/ui/Menu.qml
@@ -55,7 +55,7 @@ PlasmaExtras.ScrollArea {
             onAction: menu.action(uuid)
         }
 
-        Kirigami.PlaceholderMessage {
+        PlasmaExtras.PlaceholderMessage {
             id: emptyHint
 
             anchors.centerIn: parent
diff --git a/applets/notifications/package/contents/ui/FullRepresentation.qml b/applets/notifications/package/contents/ui/FullRepresentation.qml
index b34dee7af..989cb4341 100644
--- a/applets/notifications/package/contents/ui/FullRepresentation.qml
+++ b/applets/notifications/package/contents/ui/FullRepresentation.qml
@@ -585,7 +585,7 @@ PlasmaComponents3.Page {
                     }
                 }
 
-                Kirigami.PlaceholderMessage {
+                PlasmaExtras.PlaceholderMessage {
                     anchors.centerIn: parent
                     width: parent.width - (units.largeSpacing * 4)
 
-- 
2.27.0

